staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -97,17 +97,17 @@ func ObjectReaction(tracker ObjectTracker) ReactionFunc {
 			if err != nil {
 				return true, nil, err
 			}
-			if action.GetSubresource() == "" {
-				err = tracker.Create(gvr, action.GetObject(), ns)
-			} else {
-				// TODO: Currently we're handling subresource creation as an update
-				// on the enclosing resource. This works for some subresources but
-				// might not be generic enough.
-				err = tracker.Update(gvr, action.GetObject(), ns)
```

Comment:
I imagine removing this could introduce regression for other subresources that depend on the update behavior. How about we special case `eviction`?

nit: instead of not saving the eviction subresource and leaving the pod as it is, it seems that a successful eviction POST request should result in a pod deletion, according to https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#the-eviction-api

Reply:
> removing this could introduce regression for other subresources that depend on the update behavior.

+1. for other trival subresources like `status`, there's no "create" operation. eviction should be handled as a special case

Reply:
Where should that be handled though, in the `fakeset`? Maybe here it should return an error when attempting to create a subresource, since there is no way to handle it correctly.

Reply:
My original thought was something like

```
if action.GetSubresource() == "eviction" {
    return true, action.GetObject(), nil
}
```

However, I queried the discovery doc from a 1.15 cluster and below are the subresources that support create operation (there might be more, e.g. alpha features). I don't see any of them depend on the update behavior, since none of them takes a parent object (node/pod) as the body parameter.
```
,v1,nodes/proxy
,v1,pods/attach
,v1,pods/binding
,v1,pods/eviction
,v1,pods/exec
,v1,pods/portforward
,v1,pods/proxy
,v1,services/proxy
```

@bouk Could you fix the bazel build so we can see the test results from CI?

Reply:
```
panic: interface conversion: runtime.Object is *v1.Pod, not *v1.Binding
```

It looks like `Binding` is broken: https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/80426/pull-kubernetes-bazel-test/1154082435609137152